.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+end_of_line = 1f
+insert_final_newline = true
+charset = utf-8
+
+[*.{css,html,js,md,rb,sh,yaml,yml}]
+indent_style = space
+indent_size = 2
+
+[Makefile]
+indent_style = tab
```

Comment:
Ironically, missing an end-of-file line feed here :P

Reply:
🙃